src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -297,25 +290,20 @@ public override void Prepare(ref RecordingState state)
                     }
 
                     var recordingFileStream = new FileStream(_filePath, FileMode.Create);
-                    //Todo to use zip
-                    var gZiprecordingFileStream = new GZipStream(recordingFileStream, CompressionMode.Compress);
+                    var gZipRecordingFileStream = new GZipStream(recordingFileStream, CompressionMode.Compress);
```

Comment:
Put in using block

Reply:
The stream is used for a long run.
but I passed its creation to EnabledRecordingState that respond to its dispose and wrapped the init code in try_catch. Is it make sense?